src/main/java/core/basesyntax/strategy/PurchaseHandler.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.db.FruitTransaction;
```

Comment:
The import statement for `FruitTransaction` is not used in the current context of `PurchaseHandler`. If `FruitTransaction` is not used elsewhere in the package, consider removing this import to keep the code clean.